apps/picsa-tools/option-tool/src/app/pages/home/home.component.ts

```diff
@@ -27,13 +28,35 @@ export class HomeComponent implements OnDestroy {
   private editorIndex = 0;
   private componentDestroyed$ = new Subject();
 
+  public status = '';
+  public disabled = false;
+  public shareCode: string;
```

Comment:
nit(non-blocking) - I think we are currently only sharing as picture not code so you can probably remove this